### PR TITLE
Full integration test Skeleton

### PR DESCRIPTION
**Pull request by zrm22**
## Description

Skeleton for a full Integration test for PHG 2.1.  I also added in some simple CLI based unit tests to verify the required parameters are in fact required.

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
Added in the starting point for the Integration Test.
```

### CODE REVIEW

File: src/main/kotlin/net/maizegenetics/phgv2/cli/BuildMafVcf.kt [hunk 1]
```diff
@@ -1,16 +1,48 @@
...
-import com.github.ajalt.clikt.parameters.arguments.argument
-import com.github.ajalt.clikt.parameters.arguments.default
 import com.github.ajalt.clikt.parameters.options.default
 import com.github.ajalt.clikt.parameters.options.option
-import com.github.ajalt.clikt.parameters.types.int
+import com.github.ajalt.clikt.parameters.options.validate
 
 class BuildMafVcf : CliktCommand() {
 
+    //--maf-dir /my/maf/files -o /path/to/vcfs
```
Comment (tcasstevens) [inline src/main/kotlin/net/maizegenetics/phgv2/cli/BuildMafVcf.kt:10]: Do you want to keep this?

Comment (zrm22) [reply src/main/kotlin/net/maizegenetics/phgv2/cli/BuildMafVcf.kt:10]: Nope I meant to remove.  I will check the other ones.

File: src/main/kotlin/net/maizegenetics/phgv2/cli/BuildRefVcf.kt [hunk 2]
```diff
@@ -1,16 +1,42 @@
...
-import com.github.ajalt.clikt.parameters.types.int
+import com.github.ajalt.clikt.parameters.options.validate
+
 
 class BuildRefVcf : CliktCommand() {
 
+    //--bed /my/bed/file.bed --reference /my/ref.fasta -o /path/to/vcfs
+
+    val bed by option(help = "BED file")
+        .default("")
```
Comment (tcasstevens) [inline src/main/kotlin/net/maizegenetics/phgv2/cli/BuildRefVcf.kt:14]: Shouldn't these just be required()?

File: src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt [hunk 3]
```diff
@@ -1,16 +1,30 @@
...
-import com.github.ajalt.clikt.parameters.arguments.default
 import com.github.ajalt.clikt.parameters.options.default
 import com.github.ajalt.clikt.parameters.options.option
-import com.github.ajalt.clikt.parameters.types.int
+import com.github.ajalt.clikt.parameters.options.validate
+
 
 class LoadVcf : CliktCommand() {
 
+    val vcfDir by option(help = "VCF file directory")
```
Comment (tcasstevens) [inline src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt:11]: Seems like most these parameters should just be required()?

File: src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt [hunk 4]
```diff
@@ -1,16 +1,30 @@
...
+        .default("")
+        .validate {
+            require(it.isNotBlank()) {
+                "--vcf-dir must not be blank"
+            }
+        }
+
+    val dbPath by option(help = "Tile DB URI")
+        .default("")
```
Comment (tcasstevens) [inline src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt:20]: No default or validate needed

**Review by tcasstevens — APPROVED** [review]

File: src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt (hunk 3 above)
Comment (zrm22) [reply src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt:11]: Yes, I will remove the defaults from everything that is required.

**Review by lynnjo — APPROVED** [review]

**Pull request merged**